libs/barista-components/stacked-bar-chart/src/stacked-bar-chart.scss

```
@@ -0,0 +1,247 @@
+    display: none;
+  }
+
+  .dt-stacked-bar-chart-track {
+    min-height: 1px;
+    height: var(--dt-stacked-bar-chart-max-bar-size);
+  }
+
+  .dt-stacked-bar-chart-slice {
+    width: var(--dt-stacked-bar-chart-length);
+  }
+}
+
+.dt-stacked-bar-chart-column {
+  justify-items: center;
+  .dt-stacked-bar-chart-track-label {
+    align-self: center;
+  }
+
+  .dt-stacked-bar-chart-track-label,
+  .dt-stacked-bar-chart-track {
+    grid-row: 1;
+  }
+
+  .dt-stacked-bar-chart-track {
+    flex-direction: column-reverse;
+    max-width: var(--dt-stacked-bar-chart-max-bar-size);
+    width: 100%;
+  }
+
+  .dt-stacked-bar-chart-slice {
+    height: var(--dt-stacked-bar-chart-length);
+  }
+
+  .dt-stacked-bar-chart-axis-series {
+    grid-column: 1/-1;
+    border-bottom: 1px solid $axis-color;
+    width: 100%;
+    grid-row: 1;
+  }
+}
+
+/** Bar */
+.dt-stacked-bar-chart-bar {
+  grid-template-columns: auto 1fr;
+  @include gridPosition('grid-column', 1, 2);
+}
+
+.dt-stacked-bar-chart-bar.dt-stacked-bar-chart-label-none {
+  gap: $gap;
+  grid-template-columns: 1fr;
+  @include gridPosition('grid-column', 0, 1);
+}
+
+/** Column */
+.dt-stacked-bar-chart-column {
+  grid-template-rows: 1fr auto;
+  @include gridPosition('grid-row', 2, 1);
+  grid-template-columns: repeat(var(--dt-stacked-bar-chart-track-amount), 1fr);
+
+  .dt-stacked-bar-chart-track {
+    grid-column: var(--dt-stacked-bar-chart-track-index);
+  }
+}
+
+.dt-stacked-bar-chart-column.dt-stacked-bar-chart-label-none {
+  gap: 0;
+  grid-template-rows: 1fr;
+}
+
+/** Track */
+.dt-stacked-bar-chart-track {
+  display: flex;
+}
+.dt-stacked-bar-chart-track-background {
+  background: $track-color;
+}
+
+.dt-stacked-bar-chart-track-selectable {
+  .dt-stacked-bar-chart-slice {
+    cursor: pointer;
+  }
+}
+
+.dt-stacked-bar-chart-slice {
+  background: var(--dt-stacked-bar-chart-color);
+  box-sizing: content-box;
+  transition: width $hidden-transition-time, height $hidden-transition-time,
+    opacity $hover-transition-time;
+}
+
+.dt-stacked-bar-chart-track-hoverable {
+  .dt-stacked-bar-chart-slice:hover {
+    opacity: $hover-opacity;
+  }
+}
+
+.dt-stacked-bar-chart-slice-selected {
+  cursor: default;
+  position: relative;
+
+  &::before {
+    content: ' ';
+    display: block;
+    position: absolute;
+    border: $selected-border-size solid $selected-border-color;
+    border-radius: 3px;
+
+    .dt-stacked-bar-chart-column & {
+      box-shadow: $selected-size 0 $selected-color inset,
+        -$selected-size 0 $selected-color inset;
+      right: -$selected-size;
+      left: -$selected-size;
+      top: 0;
+      bottom: 0;
+    }
+
+    .dt-stacked-bar-chart-bar & {
+      box-shadow: 0 $selected-size $selected-color inset,
+        0 #{-$selected-size} $selected-color inset;
+      border: $selected-border-size solid $selected-border-color;
+      top: -$selected-size;
+      bottom: -$selected-size;
+      right: 0;
+      left: 0;
+    }
```

Selectors that use a `&` nesting at the end are really hard to read. Could we refactor that one?

```suggestion
.dt-stacked-bar-chart-slice-selected::before .dt-stacked-bar-chart-column,
.dt-stacked-bar-chart-slice-selected::before .dt-stacked-bar-chart-bar {
  box-shadow: 0 $selected-size $selected-color inset,
    0 #{-$selected-size} $selected-color inset;
  border: $selected-border-size solid $selected-border-color;
}

.dt-stacked-bar-chart-slice-selected::before .dt-stacked-bar-chart-column {
  right: -$selected-size;
  left: -$selected-size;
  top: 0;
  bottom: 0;
}

.dt-stacked-bar-chart-slice-selected::before .dt-stacked-bar-chart-bar {
  top: -$selected-size;
  bottom: -$selected-size;
  right: 0;
  left: 0;
}
```